vpp-manager/vpp_runner.go

```diff
@@ -289,41 +315,41 @@ func (v *VppRunner) addExtraAddresses(addrList []netlink.Addr, extraAddrCount in
 			Mask: addr.Mask,
 		}
 		a.IP[2] += byte(i)
-		err = v.vpp.AddInterfaceAddress(config.DataInterfaceSwIfIndex, a)
+		err = v.vpp.AddInterfaceAddress(vppIfSwIfIndex, a)
 		if err != nil {
 			log.Errorf("Error adding address to data interface: %v", err)
 		}
 	}
 	return nil
 }
 
-func (v *VppRunner) configureVpp() (err error) {
+func (v *VppRunner) configureVpp(idx int) (err error) {
 	// Always enable GSO feature on data interface, only a tiny negative effect on perf if GSO is not
```

Comment:
I think it's worth defining variables for the interface spec and config here rather than using the index everywhere.